RevenueCatUI/Views/FooterView.swift

```diff
@@ -147,10 +148,10 @@ private struct SeparatorView: View {
             .accessibilityHidden(true)
     }
 
-    @ScaledMetric(relativeTo: .caption)
```

Comment:
Oh thanks 👏🏻 
Unfortunately this can't use a common constant so it has to be hardcoded again.